specification/basic_concepts/basic_concepts.tex

```diff
@@ -29,38 +29,31 @@ \chapter{Basic concepts}\label{sec:basic_concepts}
-are constant and easily predictable, which is paramount for hard real-time and safety-critical applications.
+are constant and easily predictable.
 
 On top of the standard data types, UAVCAN defines a set of standard high-level functions including:
-node health monitoring, network discovery, time synchronization, firmware update,
+node health monitoring, node discovery, time synchronization, firmware update,
 plug-and-play node support, and more.
 For more information see chapter \ref{sec:application_layer}.
 
 Serialized message and service data structures are exchanged by means of the transport
-layer (chapter \ref{sec:transport_layer}),
-which implements automatic decomposition/reassembly of long transfers into/from several transport
-frames\footnote{Here and elsewhere, a \emph{transport frame} means a block of data
-that can be atomically exchanged over the network, e.g., a CAN frame.},
-allowing nodes to exchange data structures of arbitrary size.
+layer (chapter \ref{sec:transport_layer}), which implements automatic decomposition of
+long transfers into several transport frames\footnote{Here and 
+elsewhere, a \emph{transport frame} means a block of data that can be atomically exchanged 
+over the network, e.g., a CAN frame.} and reassembly from these transport frames
+back into a single transfer buffer, allowing nodes to exchange data structures of 
+arbitrary size.
 
 \begin{figure}[hbt]
     \centering
-    \setlength\arrayrulewidth{1pt}  % https://tex.stackexchange.com/a/256732/132781
-    \begin{tabularx}{0.8\textwidth}{|X|X|X|X|}
-        \hline
-        \multicolumn{4}{|c|}{Application} \\
-        \hline
-
-        & Standard functions & \multicolumn{2}{c|}{\multirow{3}{*}{Vendor-specific data types (optional)}} \\\cline{2-2}
-        \multicolumn{2}{|c|}{\multirow{2}{*}{Standard data types}} & \multicolumn{2}{c|}{} \\
-        \multicolumn{2}{|c|}{} & \multicolumn{2}{c|}{} \\
-
-        \hline
-        \multicolumn{4}{|c|}{Serialization} \\
-        \hline
-        \multicolumn{4}{|c|}{Transport layer} \\
+    \begin{tabular}{|c|c|l|c|l|c|}
         \hline
-    \end{tabularx}
+        \multicolumn{6}{|c|}{Applications}                                                                                              \\ \hline
+        \multicolumn{1}{|l|}{} & Required Functions &            & Standard Functions          &            & Custom Functions          \\ \cline{2-2} \cline{4-4} \cline{6-6} 
+        \multicolumn{2}{|c|}{Required Data Types}   & \multicolumn{2}{c|}{Standard Data Types} & \multicolumn{2}{c|}{Custom Data Types} \\ \hline
+        \multicolumn{6}{|c|}{Serialization}                                                                                             \\ \hline
+        \multicolumn{6}{|c|}{Transport}                                                                                                 \\ \hline
+    \end{tabular}
```

Comment:
Not to be nitpicking, but let us consider consistency issues. Everywhere else in the document, `Sentence case` is preferred rather than `Title Case`, including headings and tables. I'm not much into UX stuff, so correct me if I'm wrong, but Title Case seems to be a thing of the American flavor of English. Should we migrate the whole document to adopt that convention? I am weakly against that, and, frankly, I don't think I have much room to speak here as my knowledge of English is imperfect at best.

Reply:
Not a nit-pick at all. I'm always unsure which case to use in technical writing but the suggestion that is more "American" makes me agree that we should _not_ use it (The last thing we need is more bombast in this world). I'll fix.